annsrcs/ensembl/dictyostelium_discoideum

```diff
@@ -0,0 +1,20 @@
+software.name=protists
+software.version=42
```

Comment:
Shouldn't this be 43? Should we add this to ongoing move to E96 EG43 or do you need to use it as EG42 before merging that?